politeiawww/user.go

```diff
@@ -113,3 +116,24 @@ func (b *backend) ProcessUserDetails(ud *v1.UserDetails, isCurrentUser bool, isA
 
 	return &udr, nil
 }
+
+// ProcessEditUser edits a user's preferences.
+func (b *backend) ProcessEditUser(eu *v1.EditUser, user *database.User) (*v1.EditUserReply, error) {
+	if eu.MyProposalNotifications != nil {
```

Comment:
Do you need some type of validation here to make sure that the passed in EditUser object contains valid notification settings?

Reply:
I don't think we need any validation. It's just a number that represents bit flags, so the only thing a user could do is set a number that applies to bit flags that haven't been added yet.

Reply:
Right, but shouldn't we let the user know if they're trying to set bits that don't correspond to any notification settings?